frontend/public/components/dashboard/dashboards-page/cluster-dashboard/health-item.tsx

```diff
@@ -226,11 +227,21 @@ export const PrometheusHealthItem = withDashboardResources<PrometheusHealthItemP
   },
 );
 
-export const ResourceHealthItem: React.FC<ResourceHealthItemProps> = ({ subsystem }) => {
+export const ResourceHealthItem: React.FC<ResourceHealthItemProps> = ({ subsystem, namespace }) => {
   const { title, resources, healthHandler, popupComponent, popupTitle } = subsystem;
   const resourcesResult = useK8sWatchResources(resources);
-
-  const healthState = healthHandler(resourcesResult);
+  const filteredResourcesResult =
```

Comment:
can't we get data/watch here `useK8sWatchResources` based on namespace as we already know that, we can avoid this. WDYT?

Reply:
made resources to include namespace. This avoids the filter. Thanks for the catch.